src/librustc/diagnostics.rs

```diff
@@ -772,6 +776,24 @@ struct Foo<'a> {
     x: &'a str,
 }
 ```
+
+Implementations need their own lifetime declarations:
+    
+```
```

Comment:
Since these code blocks stand on their own, can you make sure to tag them with `compile_fail,E0261` like the other failing sample in this excerpt? You'll also need to include the struct definition again.